code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -201,3 +201,142 @@
 		var/turf/T = loc
 		T.Entered(src)
 //Ayy lmao
+
+// IPC Emotes.

```

Comment:
Most of these are made for humans only in your code, but there are also versions for cyborgs, most of which are just duplicated here. Would be cleaner if you made the versions for cyborgs function for both them and IPCs.

Reply:
Unfortunately, without the ability to define a mob past `/mob/living/carbon/human` (for example `/mob/living/carbon/human/species/ipc`) it makes me use the hacky solution that imsxz used for *squish. I tested it, and humans and other organics can't use the IPC emotes. 

I'd likely need to grab [this](https://github.com/tgstation/tgstation/blob/aac6b076c704ddafa408f4b67edf5d1a6d41be02/code/modules/mob/living/carbon/human/human.dm#L912) from further downstream.

Once we get that would probably be as easy as putting it here: https://github.com/OracleStation/OracleStation/blob/master/code/modules/mob/living/silicon/robot/emote.dm#L2

And I can erase this entire section.

Reply:
Update: I attempted the above, it did not work. Back to the drawing board.